README-FIPS.md

```diff
@@ -1,20 +1,39 @@
 OpenSSL FIPS support
 ====================
 
-This release of OpenSSL includes a cryptographic module that is intended to be
+This release of OpenSSL includes a cryptographic module that can be
 FIPS 140-2 validated. The module is implemented as an OpenSSL provider.
 A provider is essentially a dynamically loadable module which implements
 cryptographic algorithms, see the [README-PROVIDERS](README-PROVIDERS.md) file
 for further details.
 
-The OpenSSL FIPS provider comes as shared library called `fips.so` (on Unix)
+A cryptographic module is only FIPS valid if it has gone through the complex
+FIPS 140 validation process. As this process takes a very long time, it is not
+possible to validate every minor release of OpenSSL.
+If you want to be FIPS validated then you must ONLY generate a FIPS provider
```

Comment:
Perhaps: `If you need FIPS compliance then ....`

Reply:
As you probably know the word 'compliance' in relation to a Cryptographic Module - means it is able to be validated. So I was trying to avoid using that confusing term..

Reply:
IMO it has two meanings - the module is compliant if it is able to be validated but an application is compliant if it uses validated crypto module according to the security policy.

Another option: `If you want to use a validated FIPS module then ...`

Reply:
Updated the text - see what you think now..